Extension/src/LanguageServer/client.ts

```diff
@@ -134,7 +134,65 @@ function collectSettings(filter: (key: string, val: string, settings: vscode.Wor
         }
         if (filter(key, val, settings)) {
             previousCppSettings[key] = val;
-            result[key] = (key === "clang_format_path") ? "..." : String(previousCppSettings[key]);
+            switch (key.toLowerCase()) {
+                case "clang_format_path": {
```

Comment:
I think we should just remove clang_format_path from telemetry

Reply:
perhaps the above if-expression should be `if (filter(key, val, settings) && isTelemetryAllowed(key, val))` and then move all of the special cases to a new function `isTelemetryAllowed` and filter them there so that this function can remain simple.

Reply:
What do you mean by "remove clang_format_path from telemetry"? Do you mean just use "..." like we were doing before?

Reply:
I meant to just filter it out completely (never send that property)

Reply:
Okay, I stopped sending it.